src/main/java/cuchaz/enigma/gui/DecompiledClassSource.java

```diff
@@ -55,10 +54,10 @@ private String remapToken(Deobfuscator deobfuscator, Token token, Token movedTok
 				highlightToken(movedToken, TokenHighlightType.DEOBFUSCATED);
 				return translatedEntry.getSourceRemapName();
 			} else {
-				String proposedName = proposeName(deobfuscator, entry);
-				if (proposedName != null) {
+				Optional<String> proposedName = proposeName(deobfuscator, entry);
+				if (proposedName.isPresent()) {
 					highlightToken(movedToken, TokenHighlightType.PROPOSED);
-					return proposedName;
+					return proposedName.get();
```

Comment:
Do not use `get` on an Optional. This body should be using `map` and `orElse` calls.

Reply:
An `isPresent` check is done in the second line above.